CMakeLists.txt

```diff
@@ -451,29 +448,51 @@ else()
     util/io_posix.cc)
 endif()
 
+if(WIN32)
+  set(SYSTEM_LIBS ${SYSTEM_LIBS} Shlwapi.lib Rpcrt4.lib)
+  set(ROCKSDB_STATIC_LIB rocksdblib${ARTIFACT_SUFFIX})
+  set(ROCKSDB_SHARED_LIB ${ROCKSDB_STATIC_LIB})
+  set(ROCKSDB_IMPORT_LIB rocksdb${ARTIFACT_SUFFIX})
```

Comment:
I was performing this mechanical transformation:

```
rocksdblib${ARTIFACT_SUFFIX} -> ROCKSDB_STATIC_LIB
rocksdb${ARTIFACT_SUFFIX} -> ROCKSDB_IMPORT_LIB
```

so the entire change is a no-op on Windows. But I don't understand the platform well enough to make changes and debugging via appveyor.yml is not the most efficient.

If you could comment with specific changes, I'm happy to put it in here and resubmit.